internal/params/config.go

```diff
@@ -290,8 +295,9 @@ var (
 		LeaderRotationExternalValidatorsEpoch: big.NewInt(6),
 		FeeCollectEpoch:                       big.NewInt(2),
 		ValidatorCodeFixEpoch:                 big.NewInt(2),
-		HIP30Epoch:                            EpochTBD,
+		HIP30Epoch:                            big.NewInt(3),
```

Comment:
It will never pass tests with such condition. We need to remove it from localnet, or create additional command in Makefile to start localnet with different options.